---
float32[6] curforce